### PR TITLE
[IN-69] Setup database infrastructure

### PR DESCRIPTION
**Pull request by jimmy-martin**
Linked to this ticket: https://instamint-deva.atlassian.net/browse/IN-69

### CODE REVIEW

File: prisma/schema.prisma
Comment (adamjorge) [inline prisma/schema.prisma:1]: Remove commented code here

**Review by adamjorge — CHANGES_REQUESTED** [review]
Overall, it seems good to me. The database conceptual data model appears to be respected. Just remove the auto-generated commented code at the start of the Prisma schema.

**Review by adamjorge — APPROVED** [review]
Commented code got removed, ok to merge

File: prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  url      = env("DATABASE_URL")
+}
+
+model Minter {
+  id                  Int       @id @default(autoincrement())
+  username            String    @unique
+  email               String    @unique
+  password            String
+  phoneNumber         String?
+  defaultLanguage     String    @default("fr")
```
Comment (avetisk) [inline prisma/schema.prisma:16]: Why not just `language`?

File: prisma/schema.prisma [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  name      String   @unique
+  createdAt DateTime @default(now())
+
+  nfts Nft[]
+}
+
+model Comment {
+  id         Int      @id @default(autoincrement())
+  content    String   @db.Text
+  isReported Boolean  @default(false)
```
Comment (avetisk) [inline prisma/schema.prisma:83]: By whom? About what?

File: prisma/scripts/comment/create-comments.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { PrismaClient } from "@prisma/client"
+
+export async function createComments(prisma: PrismaClient) {
```
Comment (avetisk) [inline prisma/scripts/comment/create-comments.ts:3]: Should be called seed.

File: prisma/seed.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+const prisma = new PrismaClient()
+
+async function main() {
+  await createAdmins(prisma)
+  await createMinters(prisma)
+  await createHashtags(prisma)
+  await createOriginalContents(prisma)
+  await createNfts(prisma)
+  await createComments(prisma)
```
Comment (avetisk) [inline prisma/seed.ts:19]: What the use of splitting your seeds?

**Pull request merged**